lua/lazy/view/float.lua

```diff
@@ -53,7 +53,7 @@ function M:init(opts)
   self.opts = vim.tbl_deep_extend("force", {
     size = Config.options.ui.size,
     style = "minimal",
-    border = Config.options.ui.border or "none",
+    border = Config.options.ui.border or vim.o.winborder or "none",
```

Comment:
This will not work properly with winborder now supporting custom borderchars. See here for example for how it can be supported: https://github.com/Saghen/blink.cmp/pull/1984